### PR TITLE
Adds SharePoint marketplace guidance

### PR DESCRIPTION
**Pull request by waldekmastykarz**
## Category

- [ ] Content fix
- [x] New article

## What's in this Pull Request?

New guidance about publishing SPFx solutions to the marketplace

### CODE REVIEW

File: docs/spfx/publish-to-marketplace.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+## Install applications from the marketplace in the tenant app catalog
+
+SharePoint Online allows you to create a tenant app catalog where you centrally manage applications available in your organization. Additionally, you can create site collection app catalogs, where you can deploy applications to specific sites.
+
+Applications that you acquire from the marketplace, you can install only in tenant app catalogs. After installing the application, you can decide whether you want to deploy it globally and make it available to all sites in your tenant or if you want to deploy it only to specific sites.
+
+For more information about managing SharePoint Framework solution in the app catalog visit: [https://docs.microsoft.com/en-us/sharepoint/use-app-catalog](https://docs.microsoft.com/en-us/sharepoint/use-app-catalog) and [https://docs.microsoft.com/en-us/sharepoint/dev/general-development/site-collection-app-catalog](https://docs.microsoft.com/en-us/sharepoint/dev/general-development/site-collection-app-catalog).
```
Comment (andrewconnell) [inline docs/spfx/publish-to-marketplace.md:40]: Need to remove locale from URLs... let MSDocs redirect as appropriate for user.

File: docs/spfx/publish-to-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+By default, only tenant administrators can install applications from the marketplace. They can however choose to delegate the responsibility to others in your organization by granting them access to the tenant app catalog.
+
+SharePoint Framework applications execute on behalf of the current user and can do whatever the current user can do. Because of this, only tenant app catalog administrators can install applications from the marketplace. Users who are not app catalog administrators can request a specific application to be installed by the app catalog administrator, but they cannot install the application themselves.
+
+## Use auto-generated Microsoft Teams manifest
+
+When building SharePoint Framework web parts, you can choose to [expose them in SharePoint and Microsoft Teams as personal apps or tabs](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/integrate-with-teams-introduction). To use your web parts in Teams you can either generate the manifest and the Teams solution package yourself or have the SharePoint Framework generate it for you. Creating the manifest manually gives you more flexibility and allows you to create multi-tab personal apps.
```
Comment (andrewconnell) [inline docs/spfx/publish-to-marketplace.md:50]: Remove locale from URL

File: docs/spfx/publish-to-marketplace.md (hunk 1 above)
Comment (waldekmastykarz) [reply docs/spfx/publish-to-marketplace.md:40]: I saw the notification, but wasn't aware you could just drop the locale. Thanks for the tip! I'll update it!

**Comment by VesaJuvonen** [comment]
thx

**Pull request merged**